### PR TITLE
1476 object does not exist exception

### PR DESCRIPTION
**Pull request by levBagryansky**
#1476
Print human-readable message if object isn't found

### CODE REVIEW

**Comment by levBagryansky** [comment]
@volodya-lombrozo please review

File: eo-runtime/src/test/java/org/eolang/MainTest.java [hunk 1]
```diff
@@ -84,6 +84,14 @@ public void jvmFullRunWithError() throws Exception {
         );
     }
 
+    @Test
+    public void objectNotFoundException() throws Exception {
+        MatcherAssert.assertThat(
+            MainTest.exec("--verbose", "unavailable-name"),
```
Comment (volodya-lombrozo) [inline eo-runtime/src/test/java/org/eolang/MainTest.java:90]: @levBagryansky should we use "--verbose" flag to print "Can not find \"unavailable-name\" object"?  Or the compiler will print the message anyway?

File: eo-runtime/src/main/java/org/eolang/Main.java [hunk 2]
```diff
@@ -177,10 +177,17 @@ private static void run(final List<String> opts) throws Exception {
...
+        final Phi app;
+        try {
+            app = Phi.class.cast(
+                Class.forName(path).getConstructor(Phi.class)
                 .newInstance(Phi.Φ)
-        );
+            );
+        } catch (final ClassNotFoundException ex) {
+            throw new ExUnset(
+                String.format("Can not find \"%s\" object", opts.get(0))
```
Comment (volodya-lombrozo) [inline eo-runtime/src/main/java/org/eolang/Main.java:188]: @levBagryansky what do you think if we write message using single quotes? 
```java
String.format("Can not find '%s' object", opts.get(0))
```
In that case we don't need to escape "

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]

File: eo-runtime/src/main/java/org/eolang/Main.java (hunk 2 above)
Comment (levBagryansky) [reply eo-runtime/src/main/java/org/eolang/Main.java:188]: @volodya-lombrozo ok

File: eo-runtime/src/test/java/org/eolang/MainTest.java (hunk 1 above)
Comment (levBagryansky) [reply eo-runtime/src/test/java/org/eolang/MainTest.java:90]: @volodya-lombrozo Indeed, it is printed anyway

**Review by volodya-lombrozo — APPROVED** [review]
Looks good to me. @yegor256 merge

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**